libreoffice-24.8.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: libreoffice-24.8
-  version: 24.8.3.1
+  version: 24.8.3.2
```

Comment:
this needs to remain the same, when making changes to the melange yaml we need to bump the epoch below.